Spigot-API-Patches/0218-Add-Player-teleportRelative-and-Player-teleportWitho.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Anrza <andrzejrzeczycki314@gmail.com>
+Date: Wed, 15 Jul 2020 12:29:19 +0200
+Subject: [PATCH] Add Player#teleportRelative and
+ Player#teleportWithoutRotation
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 2eb3121386e3bc5ccdd74726a7c4b5f832ec5aea..e98b5c3e66eb175dc6bdbdbdb8e6f3d8ea194ead 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1770,6 +1770,19 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     @NotNull
+     <T> T getClientOption(@NotNull ClientOption<T> option);
++
++    /**
++     * Teleports the player without forcing rotation.
++     * @param location Destination location. Yaw and pitch will be ignored.
++     */
++    void teleportWithoutRotation(@NotNull Location location);
```

Comment:
This feels best left to plugins, as they can easily just do getPitch and .getYaw themselves.